acceptance/framework/helpers/helpers.go

```diff
@@ -150,3 +150,37 @@ func MergeMaps(a, b map[string]string) {
 		a[k] = v
 	}
 }
+
+// RegisterExternalService registers an external service with Consul for testing purposes.
+// The function creates a Consul service registration for the specified service, optionally within a namespace.
+// If a namespace is provided, it creates the namespace in Consul and associates the service with it.
+func RegisterExternalService(t *testing.T, consulClient *api.Client, namespace, name string) {
```

Comment:
I moved `RegisterExternalService` out of `terminating-gateway` so that it can more easily be used by other tests.